lib/Agrammon/Test.pm6

```diff
@@ -0,0 +1,43 @@
+
+## THIS FILE TO BE REMOVED AS SOON AS Cro::HTTP::Test supplies these
```

Comment:
Hmm, I thought the PR was merged for those...

Reply:
That said, I don't think you're actually using these helpers anyway?

Reply:
Yes, I think I committed this accidentally. Removed.